cli/tools/bench/mod.rs

```diff
@@ -394,25 +394,16 @@ async fn bench_specifiers(
 }
 
 /// Checks if the path has a basename and extension Deno supports for benches.
-fn is_supported_bench_path(path: &Path, patterns: &FilePatterns) -> bool {
-  if !is_script_ext(path) {
+fn is_supported_bench_path(entry: WalkEntry) -> bool {
```

Comment:
It's a bit scary that we have to update every single subcommand to the new mechanism :D

Reply:
They're all basically doing the same thing. It's mostly a refactor from arguments to a single argument and then consolidating the duplicate custom bench & test code to the same function then handling negated globs/paths in the consolidation.